variables.tf

```diff
@@ -134,6 +134,14 @@ variable "default_cert_domain_name" {
   type        = string
 }
 
+variable "health_check" {
+  description = "Elastic Load Balancer health check configuration"
```

Comment:
It may be worth linking to some documentation (either for AWS in general or for the `aws_alb_target_group` Terraform resource specifically) about what to provide and how to structure the data for this variable.

Reply:
That makes sense. As much as is reasonable, I'm very much a fan of letting the terraform modules registry website document the inputs/outputs for us, and avoid manually documenting them in the README files (to avoid violating DRY), so I appreciate the desire to keep the README succinct.

Thanks for talking me through this.

Reply:
Unfortunately, the Terraform Registry auto-generated documentation comes from the `description` field in variable blocks, and there's very limited formatting capability in those. Last I checked, there was an issue on the Terraform GitHub repository about this. All that to say, README docs are still helpful, so thanks for the suggestion.